.github/workflows/ci_tests.yml

```
@@ -0,0 +1,86 @@
+name: CI Tests
```

Shortening it makes it easier to include a badge in README. See `astropy` for example.

```suggestion
name: CI
```